### DISCUSSION TITLE
Markdown views (notes) [Apr 20]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5556] [Discord Threads]
> 🙋‍♂️ **Hioko**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### Currently not supported. What's your use case?  
#  
> 🙋‍♂️ **Hioko**  
#### just like tana, i want to have integrated tool that can help to write notes as well as arrange in form of tabular/kanban/categorization  
#  
> 🙋‍♂️ **Hioko**  
#### i think every other tool like obsidian/logseq can also convert view into tabular form but it is quite limited as it is based on tag/label assigned.  
#  
> 🙋‍♂️ **Hioko**  
#### they are not as fluent/fluid as tana, i guess (though i ahvent used it)  
#  
> 🙋‍♂️ **Hioko**  
#### i am looking for a hybrid setup/view which can be converted from notes to table or vice-versa  
#  
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### We haven't tried Tana either. Heard about supertags from one of their promotional videos. Initial impression - it looked like lot of learning to be done before we can use that tool. Would love to hear from you about your use case and specific features that caught your attention & why you find it really useful?  
#  
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### Saw more details from other comment. will check it out  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1087079183886319668)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
